cake.cs

```diff
@@ -1,16 +1,15 @@
+#:sdk Cake.Sdk
+#:package Cake.FileHelpers
+#:package Newtonsoft.Json
+#:property ProjectType=Test
```

Comment:
This seems a bit weird to me. I know that it's _running_ tests, but it doesn't feel like tests.